### PR TITLE
Market state change extention

### PR DESCRIPTION
**Pull request by brossetti1**
wip draft pr for completion of the closed | paused | open in market status badges.

## Description
- [X] implements paused markets
- [X] implements timers for markets open in (green)
- [X] implements timers for markets closes in (red)
- [X] implements market overlay for when markets are suspended (paused or closed)
- [x] fix styling and aggregate it to `selectedheme`

## Related issue
https://github.com/Kwenta/kwenta/issues/594

## Motivation and Context
I wanted to split this out into a simple implementation as a good point to iterate on, while also providing basic functionality of paused markets. There are small UI issues still and there is a decent amount of refactoring which im working on via another branch. If that branch gets into a finished state before we need to merge this, I will include those changes here.

## How Has This Been Tested?
- forced to pasued - tested deploy on https://github.com/Kwenta/kwenta/pull/679
- local dev environment

## Screenshots (if appropriate):
Screen Shot 2022-04-13 at 3 48 40 PM
Screen Shot 2022-04-13 at 3 48 47 PM
Screen Shot 2022-04-13 at 7 51 23 AM

### CODE REVIEW

File: constants/currency.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export const MARKET_HOURS_SYNTHS = new Set([
 	...AFTER_HOURS_SYNTHS,
 ]);
 
-export const COMMODITY_SYNTHS = new Set<CurrencyKey>([]);
+export const COMMODITY_SYNTHS = new Set<any>(['AAVE', 'XAU', 'XAG', 'WTI']);
```
Comment (brossetti1) [inline constants/currency.ts:69]: demo commodity synths

File: hooks/useMarketClosed.ts [hunk 2]
```diff
@@ -17,9 +17,9 @@ const useMarketClosed = (currencyKey: CurrencyKey | null) => {
 			: false;
 
 	return {
-		isMarketClosed: isCurrencySuspended,
+		isMarketClosed: true, // isCurrencySuspended
 		isCurrencySuspended,
-		marketClosureReason: currencySuspendedQuery.data?.reason as MarketClosureReason,
+		marketClosureReason: 'market-closure' as MarketClosureReason, // currencySuspendedQuery.data?.reason as MarketClosureReason,
```
Comment (brossetti1) [inline hooks/useMarketClosed.ts:22]: forcing markets to suspended

File: hooks/useMarketClosed.ts [hunk 3]
```diff
@@ -17,9 +17,9 @@ const useMarketClosed = (currencyKey: CurrencyKey | null) => {
 			: false;
 
 	return {
-		isMarketClosed: isCurrencySuspended,
+		isMarketClosed: true, // isCurrencySuspended,
 		isCurrencySuspended,
-		marketClosureReason: currencySuspendedQuery.data?.reason as MarketClosureReason,
+		marketClosureReason: 'market-closure' as MarketClosureReason, // currencySuspendedQuery.data?.reason as MarketClosureReason,
```
Comment (Tburm) [inline hooks/useMarketClosed.ts:22]: Assuming this is what we need to remove for hard-coding?

File: components/Badge/Badge.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+	font-family: ${(props) => props.theme.fonts.black};
+	background: ${(props) => props.theme.colors.common.primaryRed};
+	border-radius: 100px;
+	letter-spacing: 1px;
+	margin-left: 4px;
+	line-height: unset;
+	font-size: 10px;
+	font-variant: all-small-caps;
+	opacity: 1;
+`;
```
Comment (Tburm) [inline components/Badge/Badge.tsx:18]:
```suggestion
	opacity: 1;
	user-select: none;
`;
Maybe this, to prevent hovering/selection
```

**Review by Tburm — APPROVED** [review]
Looks good, as long as we fix the hard-coding before release.

I tested the version with hard-coding, visually it looks the way I would expect. Don't see any bugs in data or old components.

File: hooks/useMarketClosed.ts (hunk 3 above)
Comment (brossetti1) [reply hooks/useMarketClosed.ts:22]: yeah thats right, also in the `useGetFuturesMarkets` - its this commit that details the forced markets:

https://github.com/Kwenta/kwenta/pull/692/commits/3aeafbe347bad037849d24a62808efa7d0d2ed9f

**Review by Tburm — APPROVED** [review]

**Pull request merged**